packages/web-component/src/components/read-along-component/read-along.tsx

```diff
@@ -938,6 +954,20 @@ export class ReadAlongComponent {
         .subscribe((el_tag) => {
           // Only highlight when playing
           if (this.playing) {
+            //if auto pause is active and not on last word of the read along pause the audio
+            if (
+              this.autoPauseAtEndOfPage &&
+              el_tag in this.endOfPageTags &&
+              this.finalTaggedWord !== el_tag
+            ) {
+              //wait for the audio of the last word
+              if (!this.isAutoPaused)
+                setTimeout(
+                  () => this.pause(),
+                  this.endOfPageTags[el_tag] * 800
```

Comment:
whet does this 800 constant value mean?

Reply:
Just tested, and it seems to work well, except this constant is funky: we're setting the timeout at 800ms per second of duration or the last word in the page. If we need to stop before the end of the last word, I'd rather see us doing something like `this.endOfPageTags[el_tag] * 1000 - 50` and document why we want to stop 50ms (or whatever number we choose) before the end of this last word.

Reply:
It is the best "guess estimate" to help convert the duration of the word into a timeout value. Ideally, I would convert the duration directly to milliseconds; however, this bit of code activates during the silence before the beginning of the word in the audio. Maybe Aidan knows the exact point in the silence at which the event is emitted. Then, we can use that in a formula for the timeout value.